geode-core/src/main/java/org/apache/geode/internal/cache/wan/GatewaySenderEventImpl.java

```diff
@@ -171,6 +171,8 @@
 
   protected boolean isInitialized;
 
+  public boolean isConcurrencyConflict = false;
```

Comment:
Don't make fields public. Make it private and add public gettor/settor methods

Reply:
done